blog/_posts/btcpay-server-1-8-0.md

```diff
@@ -0,0 +1,164 @@
+
+![](/images/btcpaylabelsblog.jpg)
+
+🔔 Rehauled Notifications icon set
+As the world moves on and smileys become emojis. We are adding more clear indicators for  your notifications. In a glance you now will be able to pick up if there’s an action required or if it’s just telling you there’s an invoice paid. You can find all icons and their meaning on our design pages [here](https://design.btcpayserver.org/design/icons/). 
+
+![](/images/btcpayiconsetblog.jpg)
+
+👩‍💻 Greenfield API updates
+
+Our Greenfield API keeps getting better. Leading up towards release 1.8.0 we’ve introduced some new features to reach through the API. We’ve added a new “get” endpoint for LNURL processing. Next to that we’ve also added 2 new permissions for PullPayments and added payments list as an endpoint. Find more details [here](https://github.com/btcpayserver/btcpayserver/releases/tag/v1.7.6).
+
+
+🏪 Store Branding.
+Remember how we introduced new store branding options in [1.7.0?](https://blog.btcpayserver.org/btcpay-server-1-7-0/) We are going to the next step in this release! When you set the store colour and logo, they will now be visible on all public facing pages. Custom forms, receipts, payment requests. pull payments, point of sale and crowdfunding can all now be customized by altering a simple color scheme in your store setting. 
+
+![](/images/btcpayserver-storebranding.png)
+
+🛍️ Checkout has copy addresses back! 
+We’ve heard your feedback on our V2 checkout. One of the main points that came back, and we fully acknowledge, was the request for copy on the invoice address. Going forward you’ll get to see the addresses (both on-chain and lightning if available) for the invoice right below the unified QR code.
+
+![](/images/btcpaycheckoutblog.jpg)
+
+📠 Keypad POS tracks values individually  
+Hot summers day on the terras? Got lots of invoices to handle with tips or discounts ? Within your invoices, you’ll now be able to see these amounts, in a split view on every subject. No longer do you have to count out what the tip or discounts amounts were, just a glance at the bottom of your invoice and all is clear. Find out more details about tracking values [here.](https://github.com/btcpayserver/btcpayserver/pull/4668) 
+
+🔌 Plugins are here to stay! 
+And do we kick it off with a great new one's made by Kukks. The plugin we are shipping with this release is called NIP05, verify your nostr account with your BTCPay Server! When verifying include a set of relays you often use for more easy discoverability. All set and done? Your NIP5 handle will be name@yourbtcpayserver.domain. If you have multiple domains mapped to the same BTCPay Server, they will all be valid. Find more details on the NIP05 plugin [here.](https://github.com/Kukks/BTCPayServerPlugins/tree/master/Plugins/BTCPayServer.Plugins.NIP05) 
+
+🔌 BTCPay Server X Coinjoin
```

Comment:
```suggestion
## 🔌 BTCPay Server X Coinjoin
```

Also this title kinda implies a collaboraiton between the BTCPay Server the project and CoinJoin, let's communicate so it's obvioios it's third-party created plugin in this case Kukks's